fastchat/serve/controller.py

```diff
@@ -254,6 +257,29 @@ def worker_api_generate_stream(self, params):
         except requests.exceptions.RequestException as e:
             yield self.handle_worker_timeout(worker_addr)
 
+    def bigdl_quote_generation(self, userdata):
+        if not enable_attest:
+            return {"quote": "Attestation not enabled"}
+        try:
+            quote_b = quote_generator.generate_tdx_quote(userdata)
+            quote = base64.b64encode(quote_b).decode('utf-8')
+            return {"quote": quote}
+        except Exception as e:
+            return {"quote": "quote generation failed: %s"%(e)}
+
+    def bigdl_attest_workers(self, userdata):
+        ret = []
```

Comment:
Maybe also test `enable_attest` and return useful information when attestation is not enabled?